### PR TITLE
Capturing Standardisation Learning

### PR DESCRIPTION
**Pull request by scarlett-k-nhs**
# Summary

**What/Why**

This branch adds standardisation spike research to the mkdocs.

### CODE REVIEW

File: docs/standardisation-research/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+1. Design a Generalised approach: This will be the default normalisation process, so would allow individuals to consider additional entities.
+    * Applying simple NLP techniques (lowercasing, removing punctuation) - Low Code
+    * High Code Implementation (Either Or)
+        * **Wikification** - [Wikimapper](https://pypi.org/project/wikimapper/) is likely the easiest package to implement quickly.
+            * **Limitation:**: If a value is None - it could be assigned a Null value.
+        * **String Comparison between entities**
+            * **Limitations:** You would need to determine a cut-off point to determine whether phrases are the same.
+        * **Wikification + String Comparison:** Use Wikification, and for values assigned None you could consider String Comparison for a high cut-off point.
+2. **Entity type specific:** We likely want to handle specific entities in a certain way and have a generalised way to begin with)
```
Comment (willpoulett) [inline docs/standardisation-research/index.md:81]: Is an extra )

File: docs/standardisation-research/standardising_with_knowledge_graphs.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+
+### REBEL + Llama Index:
+
+**[REBEL extracts triplets from text](https://medium.com/@sauravjoshi23/building-knowledge-graphs-rebel-llamaindex-and-rebel-llamaindex-8769cf800115):** This is chunked to ensure REBEL can extract the information out.
+
+
+### KnowledgeGraph
+
+[KnowledgeGraph](https://github.com/rahulnyk/knowledge_graph): This demonstrates a framework from going from document to graph. - the codebase would likely need reworking.
```
Comment (willpoulett) [inline docs/standardisation-research/standardising_with_knowledge_graphs.md:113]: . - should just be -

File: docs/standardisation-research/standardising_with_knowledge_graphs.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+3. **[PyVis](https://pyvis.readthedocs.io/en/latest/)** to visualise the graphs.
+
+### Graph_Maker: Requires GROQ
+
+1.  Define your own ontology i.e. your entities and a description of what those entities are.
+
+2. Run the Graph-maker using a large language model to create your graph.
+
+3. Then you can use this graph and created it over your documents.
```
Comment (willpoulett) [inline docs/standardisation-research/standardising_with_knowledge_graphs.md:127]: sentence doesn't quite make sense

**Review by willpoulett — CHANGES_REQUESTED** [review]
Really good and interesting research! Only 3 small typo-like comments.

**Review by danjscho — APPROVED** [review]

**Review by willpoulett — APPROVED** [review]

**Pull request merged**